features/tailormade_auth/build.gradle

```diff
@@ -0,0 +1,83 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'kotlin-kapt'
+    id 'kotlin-android-extensions'
+    id 'androidx.navigation.safeargs.kotlin'
+    id 'dagger.hilt.android.plugin'
+}
+
+android {
+    compileSdkVersion 30
```

Comment:
set compileSdkVersion, minSdkVersion, targetSdkVersion, versionCode, versionName in gradle project level and set in all build.gradle of each module